cve_bin_tool/cvedb.py

```diff
@@ -91,7 +92,10 @@ async def nist_scrape(self, session):
             json_meta_links = self.META_REGEX.findall(page)
             return dict(
                 await asyncio.gather(
-                    *[self.getmeta(session, meta_url) for meta_url in json_meta_links]
+                    *[
+                        self.getmeta(session, self.META_LINK + meta_url)
```

Comment:
Can we switch this to an f-string?  We've stopped using + for string manipulation.  More details here: https://github.com/intel/cve-bin-tool/blob/master/doc/CONTRIBUTORS.md#string-formatting

Reply:
But other than that, this looks like what we need. Let's just get the PR cleaned up a little bit so we can merge it without having to overwrite it immediately. :)

Reply:
I revert back the changes after switching to f-string. I think this will work now